### ISSUE TITLE
hctl shutdown reports ERROR on VM

### ISSUE THREAD

**Issue by nezzzu** [Seagate/cortx-hare#1474]
```
seagate@localhost~/1nodecluster[861]hctl shutdown
Stopping m0d@0x7200000000000001:0xc (ios) at localhost... 
Stopped m0d@0x7200000000000001:0xc (ios) at localhost
Stopping m0d@0x7200000000000001:0x9 (confd) at localhost... 
Stopped m0d@0x7200000000000001:0x9 (confd) at localhost
Stopping hare-hax at localhost... 
Stopped hare-hax at localhost
Stopping hare-consul-agent at localhost... 
Stopped hare-consul-agent at localhost
Shutting down RC Leader at localhost... 
**ERROR**
seagate@localhost~/1nodecluster[862]
```

**Comment by supritshinde** [comment]
Hi @nezzzu I'm currently trying to reproduce this thing at my side.

**Comment by knekrasov** [comment]
Related JIRA ticket https://jts.seagate.com/browse/EOS-18751

**Comment by supritshinde** [comment]
Issue resolved PR link: https://github.com/Seagate/cortx-hare/pull/1552

**Comment by s-arya** [comment]
Issue resolved PR link: #1552
